### ISSUE TITLE
option enregistrement connexion 

### ISSUE THREAD

**Issue by c4coer** [seansfkelley/nas-download-manager#202]
bonjour
Pouvez-vous mettre une option, pour enregistrer les paramètres sans test de connexion.
merci

Can you put an option, to save the connection settings without the connection test.
thanks

**Comment by seansfkelley** [comment]
Yeah, I've also been annoyed by being unable to do this. I'll try to include it next time I have some time to work on this project.

**Comment by c4coer** [comment]
thank

**Comment by seansfkelley** [comment]
I implemented this on a branch, but after looking into #190 as well, I wasn't able to come up with a good way to support this and #190 simultaneously, so I undid it. I'll reopen this issue because doing this is still desirable and I'm hoping I can come up with a way to do so that isn't really confusing to use.
